packages/menu-bar/src/vaadin-menu-bar.js

```diff
@@ -200,6 +201,30 @@ class MenuBar extends ButtonsMixin(InteractionsMixin(ElementMixin(ThemableMixin(
     };
   }
 
+  /**
+   * Override method inherited from `DisabledMixin`
+   * to update the `disabled` property for the buttons
+   * whenever the property changes on the menu bar.
+   *
+   * @param {boolean} newValue the new disabled value
+   * @param {boolean} oldValue the previous disabled value
+   * @override
+   * @protected
+   */
+  _disabledChanged(newValue, oldValue) {
+    super._disabledChanged(newValue, oldValue);
+    if (oldValue !== newValue) {
+      this.__updateItemsDisabledProperty();
+    }
+  }
+
+  __updateItemsDisabledProperty() {
```

Comment:
Please add JSDoc annotation:

```suggestion
  /** @private */
  __updateItemsDisabledProperty() {
```

Reply:
Done.